pkg/toolchains/rpmbuild.bzl

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""toolchain to provide an rpmbuild binary."""
+
+RpmbuildInfo = provider(
+    doc = """Platform inde artifact.""",
+    fields = {
+        "label": "The path to a target I will build",
+        "path": "The path to a pre-built rpmbuild",
```

Comment:
Nit: This isn't entirely accurate: in our private builds, we've been using a pre-built rpmbuild that is referenced via a label.

Granted, I needed to wrap it with a generated (built) script to get it to work, so this may be more accurate than I originally thought.